apps/demos/Demos/HtmlEditor/AITextEditing/React/App.tsx

```diff
@@ -16,7 +9,22 @@ export default function App() {
       defaultValue={markup}
       aiIntegration={aiIntegration}
     >
-      <Toolbar items={toolbarItems}></Toolbar>
+      <Toolbar>
+        <ToolbarItem name="ai">
+          <Command name="summarize" />
+          <Command name="proofread" />
+          <Command name="expand" />
+          <Command name="shorten" />
+          <Command name="changeStyle" />
+          <Command name="changeTone" />
+          <Command name="translate" />
+          <Command name="askAI" />
+          <Command {...extractKeywordsCommand} />
```

Comment:
I doubt that we should use **property binding** for name and text in this case. I even think that it would be better for the user to see immediately in the markup that this is a **custom** command. The same for Angular and Vue